Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -21,9 +47,28 @@ def model(parameters):
     sys.stdout = sys.__stdout__ # Re-enable printing
     print("Simulation completed")
     
-    # Obtain summary statistics from simulation
-    dummy_result = np.array([parameters["gamma"], parameters["reg_param"]]) #Just return parameter values until summary statistics are working
-    return {"data": dummy_result}
+    # Load cellStates
+    pickle_list = helperFunctions.create_pickle_list(export_path)
```

Comment:
this function is not necessary. You could do this just by writing one line in your ABC code. Even if you prefer to stick to this make it more professional. In the current format of this function, "if" statements have been run a lot which is not efficient and professional instead write it like : pickle_list = [pickle_file for pickle_file in stored (glob.glob(export_path + "/*.pickle))]